dashboard.py

```diff
@@ -324,6 +324,7 @@ def make_timeseries(
                     f"{spec.field_name}:Q", title=f"{spec.field_desc} {spec.unit}"
                 ),
                 altair.Tooltip("ci_run_url:N", title="CI Run URL"),
+                altair.Tooltip("cluster_id:N", title="Cluster ID"),
```

Comment:
Could `:N` be a problem for displaying the ID? this is a quantity, it should be Q
```suggestion
                altair.Tooltip("cluster_id:Q", title="Cluster ID"),
```

Reply:
It looks like the issue is just that the `cluster_id` column is filtered out above